.github/workflows/ci.yml

```diff
@@ -27,5 +28,5 @@ jobs:
 
       - name: Run tests with coverage
         run: |
-          uv pip install pytest pytest-mock pytest-cov
+          uv pip install pytest pytest-mock pytest-cov pyspark databricks-sdk
```

Comment:
this could be done via uv sync --only-group test, and have these packages specified there. Also, you do not pin these packages

Reply:
Will update that ASAP 😄